manifests/Quazar Conundrum.yaml

```
@@ -0,0 +1,11 @@
+name: The Quazar Condundrom
+authors: Anerix(Paradox)CorminQuanadol
+homepage: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/releases/tag/Major_ReleaseV015
+license: CC-BY-SA-4.0
+version: V0.1.5
+shortDescription: An Odyssey Spanning 100 years
+description: The Quazar Conundrum is a massive century spanning story involving a galxy spanning threat, changing hegemony and new factions rising to power.
+url: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/archive/refs/tags/Major_ReleaseV015.zip
+autoupdate:
+	type: tag
+	url: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/archive/refs/tags/$version.zip
```

```suggestion
  type: tag
  url: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum
```
this should fix your syntax errors - it's just whitespaces.

I was typing out a long essay about autoupdate, but you found the solution on your own. this is correct, but be aware that the first autoupdate run will replace your chosen version with `Major_ReleaseV015`. You can always name your tags differently to change that.